packages/web-components/tests/snapshots/dds-card-group.md

```diff
@@ -0,0 +1,34 @@
+# `dds-card-group`
+
+#### `Renders Default`
+
+```
+<slot>
```

Comment:
@ariellalgilmore why is the snapshot blank?

Reply:
Thanks @jeffchew!! Will update!